packages/custom-media-element/test/test.js

```diff
@@ -208,14 +208,16 @@ test('has HTMLVideoElement like properties', async function (t) {
     'videoHeight',
     'videoWidth',
     'volume',
-    'webkitAudioDecodedByteCount',
-    'webkitDecodedFrameCount',
-    'webkitDroppedFrameCount',
-    'webkitEnterFullScreen',
-    'webkitEnterFullscreen',
-    'webkitExitFullScreen',
-    'webkitExitFullscreen',
-    'webkitVideoDecodedByteCount',
+    // Commenting browser-prefixed properties out as we should not
```

Comment:
NOTE: This is unrelated but tests were failing both locally and on GHA because some webkit browser-prefixed fullscreen props were not present in the browser version used for testing.